--- /dev/null
+++ b/include/linux/hpt.h
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2010-2011	Lorenzo Bianconi <me@lorenzobianconi.net>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#ifndef _LINUX_HPT_H
+#define _LINUX_HPT_H
+extern struct timeval hpt_hook(void);
+#endif
